src/lava/proc/lif/models.py

```diff
@@ -147,8 +147,12 @@ def subthr_dynamics(self, activation_in: np.ndarray):
         # when decayed_curr is 0, we don't care about its sign. But np.mod
         # needs something non-zero to avoid the divide-by-zero warning
         sign_of_curr[sign_of_curr == 0] = 1
-        wrapped_curr = np.mod(decayed_curr,
-                              sign_of_curr * self.max_uv_val)
```

Comment:
We should probably remove line 146 to 149 now that we are not using `sign_of_curr`

Reply:
I fixed this.